organizers/2018-05-08.html

```diff
@@ -84,15 +85,35 @@ <h2>Eating and Drinking</h2>
 
 <section>
     <h1>🎺 Django News</h1>
-    <h2>Security Release</h2>
+    <h2>Bugfix release</h2>
     <ul>
-        <li>2.0.4 and 1.11.12 released last Monday</li>
-        <li>Bugfix release</li>
-        <li>Fixes crash when combining two annotated <code>values_list()</code> querysets in some circumstances.</li>
+        <li>2.0.5 and 1.11.13 released last Tuesday</li>
+        <li>Corrected the import paths that <strong>inspectdb</strong> generates for <strong>django.contrib.postgres</strong> fields</li>
+
         <li><a href="https://www.djangoproject.com/weblog/">https://www.djangoproject.com/weblog/</a></li>
     </ul>
 </section>
 
+<section>
+    <h1>🎺 Python News</h1>
+    <h2>Release</h2>
+    <ul>
+        <li>Python 2.7.15 released last Tuesday</li>
```

Comment:
Not worth mentioning? No one uses 2.7 any more! It's 2018!

Reply:
:) ok

Reply:
That was a bit of a troll, I bet you 50% of the meetup still use it, but perhaps we shouldn't encourage :P

Reply:
I heard that they drop 2.6 support. That was a big news for me but it was not on the news, I heard it in the weekly podcast I guess.